operators/operator_sdk/token_auth/osdk-cco-gcp.adoc

```diff
@@ -0,0 +1,62 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="osdk-cco-gcp"]
+= CCO-based workflow for OLM-managed Operators with {gcp-wid-short}
+
+:context: osdk-cco-gcp
+
+toc::[]
+
+When an {product-title} cluster running on {gcp-first} is in *{gcp-wid-short} / Federated Identity* mode, it means the cluster is utilizing features of {gcp-first} and {product-title} to apply permissions in {gcp-wid-short} at an application level.
+
+The Cloud Credential Operator (CCO) is a cluster Operator installed by default in {product-title} clusters running on cloud providers. Starting in {product-title} 4.17, the CCO supports workflows for OLM-managed Operators with {gcp-short}.
```

Comment:
To be precise, should be "the CCO supports workflows for OLM-managed Operators with GCP **Workload Identity**"